tests/foreman/api/test_errata.py

```diff
@@ -17,13 +17,31 @@
 :Upstream: No
 """
 # For ease of use hc refers to host-collection throughout this document
-from time import sleep
+from time import sleep, time
+from robottelo.logging import logger
 
+# from datetime import datetime
```

Comment:
if not needed, just delete
```suggestion
```

Reply:
`datetime` is used in `test_end_to_end`

Reply:
The import is commented out, datetime is not being used elsewhere in the file, and I don't see an E2E test in this file. Do a quick check to see if it should be there.

Reply:
Apologies for the confusion, I was looking at the UI `test_end_to_end`, which uses datetime but is unrelated to this pr. 
No tests in this file use it, I'll remove the for the next commit!